### DISCUSSION TITLE
Add weekly/monthly stats about progress

### DISCUSSION THREAD

**Discussion by qarmin** [dudykr/stc#409] [Ideas]
https://blog.antoyo.xyz/rustc_codegen_gcc-progress-report-16 and other rustc_codegen_gcc reports shows stats how big progress happened since last report.

I think that such stats will help user understand how big is progress of app.

Category | Last Month | This Month | Delta
-- | -- | -- | --
Passed | 4787 | 4793 | +6
Failed | 52 | 46 | -6

**Comment by kdy1** [comment]
I'm already doing it, just like https://stc.dudy.dev/blog/this-week-in-stc/week-5

**Comment by sunrabbit123** [comment]
Since this is an answered question, we'll close the session.
